docs/architecture/assets.md

```diff
@@ -1,58 +1,78 @@
-In Miden, users can create and trade arbitrary fungible and non-fungible assets.
+# Asset
 
-We differentiate between native and non-native assets in Miden. Native assets follow the Miden asset model. Non-native assets are all other data structures of value that can be exchanged.
+> Fungible, Non-fungible, Native, and Non-native assets in the Miden protocol.
```

Comment:
That might have been not very clear. Let's stick to Fungible and Non-fungible assets. There is no need to mention native and non-native assets.

The differentiation between native and non-native assets is not important for now. We can add a section below and say something like "**Non-native assets**: You don't have to follow our asset protocol, it is also possible to ..."